tests/by-util/test_cp.rs

```diff
@@ -56,6 +56,8 @@ static TEST_MOUNT_MOUNTPOINT: &str = "mount";
 static TEST_MOUNT_OTHER_FILESYSTEM_FILE: &str = "mount/DO_NOT_copy_me.txt";
 #[cfg(unix)]
 static TEST_NONEXISTENT_FILE: &str = "nonexistent_file.txt";
+#[cfg(all(unix, not(target_os = "macos")))]
```

Comment:
See my comment below.
```suggestion
#[cfg(all(unix, not(any(target_os = "android", target_os = "macos"))))]
```